src/mattermost/client.ts

```diff
@@ -0,0 +1,8 @@
+import Client from 'mattermost-redux/client/client4.js';
+const client = new Client();
+
+const url = process.env.MM_SITEURL || 'http://localhost:8065';
+client.setUrl(url);
```

Comment:
We'll definitely want to instantiate clients on the fly, as we will have user tokens and bot tokens to work with. I suggest exporting a function that accepts `(url: string, token: string)`, and returns a new `Client4`.

Reply:
yep. Thanks!

Reply:
https://github.com/mattermost/mattermost-app-zendesk/commit/be285e89847149fc6810a6312aaefb8d3574a082